pkg/util/util.go

```diff
@@ -144,3 +145,20 @@ func NormalizeWindowsPath(path string) string {
 	}
 	return normalizedPath
 }
+
+// ClearSecrets takes a request object and returns a copy of the request with
+// the "Secrets" field cleared.
+func ClearSecrets(req interface{}) interface{} {
```

Comment:
Can we add a simple unit test as we do for [other util functions](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/a8d4dae06a490a21f44e0dad59fb5543e2ab76af/pkg/util/util_test.go#L163)?

Reply:
Added ✅